### PR TITLE
NFC - minor spelling tweaks under lite/kernels directory

### PR DESCRIPTION
**Pull request by kiszk**
This PR addresses minor spelling tweaks under `tensorflow/lite/kernels` directory.
follow-on of #35286

### CODE REVIEW

**Comment by kiszk** [comment]
cc @mihaimaruseac

File: tensorflow/lite/kernels/pad_test.cc~ [hunk 1]
```diff
@@ -0,0 +1,786 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```
Comment (mihaimaruseac) [inline tensorflow/lite/kernels/pad_test.cc~]: This file should not be added. Looks like a backup file from the editor

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by renjie-liu — APPROVED** [review]

File: tensorflow/lite/kernels/pad_test.cc~ (hunk 1 above)
Comment (kiszk) [reply tensorflow/lite/kernels/pad_test.cc~]: Sorry for my mistake

**Comment by kiszk** [comment]
I may accidentally close this PR. I will recreate the PR.

**Pull request closed without merging**